src/_includes/partials/footer.html

```diff
@@ -6,11 +6,16 @@
 		<span class="footer-divider">•</span>
 		<a href="/">Events</a>
 		<span class="footer-divider">•</span>
-		<a href="/sesh">Sesh Help</a>
+		<a href="/sesh">Participate</a>
 	</div>
 
 	<div class="footer-social">
-		<a href="https://discord.com/invite/q9ZdBgP" target="_blank" rel="noreferrer noopener" aria-label="Discord">
+		<a
+			href="https://discord.com/invite/q9ZdBgP"
+			target="_blank"
+			rel="noreferrer noopener"
+			aria-label="Discord icon link"
```

Comment:
I don't think we should change this `aria-label` here, nor the GitHub or YouTube `aria-label`s. Previously, when a screenreader user navigated to this link, they'd be told `link, Discord`. Now, they'll be told `link, Discord icon link`. The previous way told them where they were going, and we didn't really need to tell them it was an icon or a link.

Reply:
oooooooh right ok that makes sense, will change back